bundles/org.openhab.binding.vallox/src/main/resources/ESH-INF/thing/se-channels.xml

```diff
@@ -0,0 +1,547 @@
+		<state min="0" max="255" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seHumiditySensor2" advanced="true">
+		<item-type>Number</item-type>
+		<label>Humidity sensor 2</label>
+		<description>Relative Humidity measured from sensor 2</description>
+		<state min="0" max="255" step="1" pattern="%d %%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2" advanced="true">
+		<item-type>Number</item-type>
+		<label>Measured Co2</label>
+		<state min="0" max="2000" step="1" pattern="%d ppm" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2High" advanced="true">
+		<item-type>Number</item-type>
+		<label>High byte of Co2 value</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2Low" advanced="true">
+		<item-type>Number</item-type>
+		<label>Low byte of Co2 value</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2SetPoint" advanced="true">
+		<item-type>Number</item-type>
+		<label>Set point for Co2 adjustment</label>
+		<state min="0" max="2000" step="1" pattern="%d ppm" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seCo2SetPointHigh" advanced="true">
+		<item-type>Number</item-type>
+		<label>High byte of Co2 set point</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2SetPointLow" advanced="true">
+		<item-type>Number</item-type>
+		<label>Low byte of Co2 set point</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seAdjustmentInterval" advanced="true">
+		<item-type>Number</item-type>
+		<label>Adjustment interval</label>
+		<description>Adjustment interval of Co2 and %RH in scale of 1-15 minutes</description>
+		<state min="1" max="15" step="1" pattern="%d min" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seAutomaticHumidityLevelSeekerState" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Automatic humidity level seeker state</label>
+		<description>Automatic humidity level seeker state</description>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seBoostSwitchMode" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Boost switch mode</label>
+		<description>Boost switch mode. OFF=Fireplace, ON=Boost</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+
+	<channel-type id="seRadiatorType" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Radiator type</label>
+		<description>Radiator type. OFF=Electric, ON=Water</description>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seCascadeAdjust" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Cascade adjust</label>
```

Comment:
I wish there was a description for this because I have no idea what this is...

Reply:
Description added.